test/shared/utils/EmptinessCheckerSpec.scala

```diff
@@ -34,11 +35,19 @@ class EmptinessCheckerSpec extends UnitSpec {
                  bar2: Option[Bar] = None)
 
   object SomeEnum {
-    case object E1 extends SomeEnum
+    given EmptinessChecker[SomeEnum] = EmptinessChecker.primitive
+  }
+
+  object Foo {
+    given EmptinessChecker[Foo] = EmptinessChecker.derived
+  }
 
-    case object E2 extends SomeEnum
+  object Bar {
+    given EmptinessChecker[Bar] = EmptinessChecker.derived
+  }
 
-    implicit val ckr: EmptinessChecker[SomeEnum] = EmptinessChecker.primitive
+  object Baz {
+    given EmptinessChecker[Baz] = EmptinessChecker.derived
```

Comment:
Please remove as they are not needed as per [here](https://github.com/hmrc/individuals-reliefs-api/commit/8d5ec9fe06ffdad860876b9e2e7d50a544d5c0c4#diff-30480b41e888e1d29899ba2b08a779f3f1eb8fd3c4210f781cc9c81dbdcc91df), thanks